components/person/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { PersonRecord } from 'types'
+import Image from 'next/image'
+import style from './style.module.css'
+export interface PeopleProps {
+	updatedList: PersonRecord[]
+}
+
+export interface AvatarProps {
+	person: PersonRecord
+}
+function Avatar({ person }: AvatarProps) {
+	if (!person.avatar || person.avatar === null) {
+		return (
+			<Image
+				className={style.avatarImage}
+				src="/profileAlt.png"
+				alt="no image for user"
+				width={105}
+				height={109}
+			/>
+		)
+	}
+	if (!person.avatar.alt) {
+		return (
+			<Image
+				className={style.avatarImage}
+				src={person.avatar.url}
+				alt="no alt name for user"
+				width={105}
+				height={109}
+			/>
+		)
+	}
+	return (
+		<Image
+			className={style.avatarImage}
+			src={person.avatar.url}
+			alt={person.avatar.alt}
+			width={105}
+			height={109}
+		/>
+	)
+}
+
+export default function Person({
+	updatedList,
+}: PeopleProps): React.ReactElement {
+	return (
+		<div className={style.peopleSection}>
+			{updatedList.length > 0 ? (
+				updatedList.map((person: PersonRecord) => (
+					<div key={person.id} className={style.peopleCube}>
+						<Avatar person={person}></Avatar>
```

Comment:
```suggestion
						<Avatar person={person} />
```
You can use the self-closing syntax if there are no `children` being passed to the component.